src/error_estimation/adjoint_refinement_estimator.C

```diff
@@ -370,93 +370,72 @@ void AdjointRefinementEstimator::estimate_error (const System & _system,
   std::unordered_set<dof_id_type> processed_node_ids;
 
   // We will be iterating over all the active elements in the fine mesh that live on
-  // this processor
-  {
-    MeshBase::const_element_iterator elem_it = mesh.active_local_elements_begin();
-    const MeshBase::const_element_iterator elem_end = mesh.active_local_elements_end();
-
-    // Start loop over elems
-    for (; elem_it != elem_end; ++elem_it)
+  // this processor.
+  for (const auto & elem : mesh.active_local_element_ptr_range())
+    for (unsigned int n=0; n != elem->n_nodes(); ++n)
```

Comment:
Huh.  Why didn't I change this to a node_ref_range()?

Reply:
Maybe this one loop didn't match your regex...